server/contexts/ctxerr/ctxerr.go

```diff
@@ -274,7 +279,8 @@ func fromContext(ctx context.Context) handler {
 }
 
 // Handle handles err by passing it to the registered error handler,
-// deduplicating it and storing it for a configured duration.
+// deduplicating it and storing it for a configured duration. It also takes
+// care of sending it to the configured APM, if any.
```

Comment:
I feel like this is ideal and will try to include a similar centralization of capturing errors when working on the Sentry ticket - we have a single `ctxerr.Handle` call that takes care of handling the error for all our needs - storage in Redis for debugging, capturing in APM and Sentry if configured. Instead of having them as distinct function calls with the same error, and the same places.

Reply:
Currently this takes care of API endpoints (in `encodeError`, regardless of whether the error is returned as part of the second return value of the endpoint or as the response's `errorer` implementation), cron jobs (the scheduler handles the errors with that call), the "worker" jobs (which are really handled as cron jobs), and the async task processing (in `service/async/async.go`, the `errHandler` is set to call this).